src/chef_workstation.js

```diff
@@ -45,14 +45,19 @@ function syncGetRegistryValue(baseKey, key, type = "REG_SZ") {
   return value;
 }
 
+function getInstallDir() {
```

Comment:
Would it make sense to locate this function in `helpers.js`?

Reply:
Maybe this one not because it is getting the installation directory of the chef-workstation gems, that is in Linux `/opt/chef-workstation` which is different from the chef-workstation-app installation directory, that is inside `/Applications/...` for Mac.

Reply:
Ah. I viewed the whole file and saw the comment documenting the purpose of this module. I see now.